packages/plugin-flex/src/commands/flex/plugins/build.ts

```diff
@@ -28,14 +26,6 @@ export default class FlexPluginsBuild extends FlexPlugin {
     await this.runScript('build');
   }
 
-  /**
-   * Parses the flags passed to this command
-   */
-  /* istanbul ignore next */
-  get _flags(): OutputFlags<typeof FlexPluginsBuild.flags> {
```

Comment:
Will this automatically inherit the base class flags? Things like `-l debug` would continue to work for the `build` command?

Reply:
Yeah it does. Verified again with flags like `-l debug` '-p=<profile>` and it is working as expected